community/dbms/src/main/java/org/neo4j/commandline/dbms/Util.java

```diff
@@ -31,6 +33,28 @@
 
 public class Util
 {
+    public static Path canonicalPath( Path path ) throws IllegalArgumentException
+    {
+        return canonicalPath( path.toFile() );
+    }
+
+    public static Path canonicalPath( String path ) throws IllegalArgumentException
+    {
+        return canonicalPath( new File( path ) );
+    }
+
+    public static Path canonicalPath( File file ) throws IllegalArgumentException
+    {
+        try
+        {
+            return Paths.get( file.getCanonicalPath() );
+        }
+        catch ( IOException e )
+        {
+            throw new IllegalArgumentException( "Unable to parse path: " + file );
```

Comment:
- Stacktraces are only thrown when `debug=true` @srbaker 
- Will fix it so the original trace is shown in that case, just as @chrisvest suggests